tests/zquantum/core/hamiltonian_test.py

```diff
@@ -303,6 +324,14 @@ def test_get_expectation_values_from_rdms_for_qubitoperator_list(
             None,
             np.array([2.0, 2.0]),
         ),
+        (
+            [
+                QubitOperator("[Z0 Z1] + [Z0] + []"),
+                QubitOperator("[X0 X1] + [X0] + 5 []"),
+            ],
+            None,
+            np.array([2.0, 2.0]),
+        ),
```

Comment:
I think we should add a case where one of the groups is just the constant term (related to my comment above)